src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    private BetDao betDao;
+
+    @Inject
+    private UserDao userDao;
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        User user;
+        Bet bet;
+
+        while (true) {
+            String input = scanner.nextLine();
+            if (input.equalsIgnoreCase("q")) {
+                return;
+            }
+
+            try {
+                String[] betAndUserData = input.split(" ");
+                String userName = betAndUserData[0];
+                int userAge = Integer.parseInt(betAndUserData[1]);
+                if (userAge < 18) {
+                    System.out.println("You are too young to place bets, go back to school!");
+                    continue;
+                }
+                user = new User(userName, userAge);
+                int value = Integer.parseInt(betAndUserData[2]);
+                double risk = Double.parseDouble(betAndUserData[3]);
+                bet = new Bet(value, risk);
+
+                userDao.addUser(user);
+                betDao.addBet(bet);
+
+                System.out.println(userDao.getAllUsers());
+                System.out.println(betDao.getAllBets());
```

Comment:
it's not necessary to display all existing users and bets every iteration of while cycle

Reply:
but mentor asked me to do that in previous part of this task)